components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
```

Comment:
I didn't tell you this yet, but we use a specific description format: all descriptions should start with "Emits an event each time...", so I'd recommend changing this to "Emits an event each time a new link is added to a subreddit".